e2eshark/run.py

```diff
@@ -1116,7 +1096,7 @@ def main():
         "--mode",
         choices=["direct", "turbine", "onnx", "ort", "vaiml"],
         default="onnx",
-        help="direct=Fx/TS->torch-mlir, turbine=integrated-e2e, onnx=exportonnx-to-torch-mlir, ort=exportonnx-to-ortep",
+        help="direct=Fx/TS->torch-mlir, turbine=aot-export->torch-mlir, onnx=exportonnx-to-torch-mlir, ort=exportonnx-to-ortep",
```

Comment:
Can you update https://github.com/nod-ai/SHARK-TestSuite/blob/main/e2eshark/README.md to talk about how to use `--mode=turbine`? (Also please feel free to restructure the file as needed, these sorts of docs get much easier to read through the more people try to follow them and propose improvements)

Reply:
Yup, will edit the README with instructions on how to use turbine mode.